scripts/records.py

```
@@ -0,0 +1,89 @@
+    page_url = page_url_from_src(src_uri, site_url)
+    return [
+        Record(
+            id=chunk_id(src_uri, chunk.section, i),
+            page_url=page_url,
+            section=chunk.section,
+            chunk_text=chunk.text,
+            embedding=emb,
+            model=model,
+            dim=dim,
+        )
+        for i, (chunk, emb) in enumerate(zip(chunks, embeddings))
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)